astroquery/mpc/tests/test_mpc_remote.py

```diff
@@ -82,17 +81,6 @@ def test_get_ephemeris_Moon_phase(self):
         result = mpc.core.MPC.get_ephemeris('2P', location='G37')
         assert result['Moon phase'][0] >= 0
 
-    def test_get_ephemeris_Uncertainty(self):
```

Comment:
What are the reason for removing these? I see that you mention to cleanup some tests to only run the mock versions but these seem to be removed for the mock test, too.

Reply:
The uncertainty column is effectively tested with the `test_get_ephemeris_unc_links` method.  Perhaps the method should be renamed.

Reply:
This is the remote file, and these are definitely the tests that need to be removed.